app/src/main/java/org/mozilla/fenix/customtabs/CustomTabsIntegration.kt

```diff
@@ -6,24 +6,50 @@ package org.mozilla.fenix.customtabs
 
 import android.app.Activity
 import android.content.Context
+import android.view.Gravity
+import android.view.View
+import androidx.coordinatorlayout.widget.CoordinatorLayout
+import androidx.core.widget.NestedScrollView
 import mozilla.components.browser.session.SessionManager
 import mozilla.components.browser.toolbar.BrowserToolbar
 import mozilla.components.feature.customtabs.CustomTabsToolbarFeature
 import mozilla.components.support.base.feature.BackHandler
 import mozilla.components.support.base.feature.LifecycleAwareFeature
+import org.mozilla.fenix.R
 import org.mozilla.fenix.components.toolbar.ToolbarMenu
 
 class CustomTabsIntegration(
     context: Context,
     sessionManager: SessionManager,
-    val toolbar: BrowserToolbar,
+    toolbar: BrowserToolbar,
     sessionId: String,
     activity: Activity?,
+    quickActionbar: NestedScrollView,
+    engineView: View,
     onItemTapped: (ToolbarMenu.Item) -> Unit = {}
 ) : LifecycleAwareFeature, BackHandler {
 
     init {
+        // Remove toolbar shadow
         toolbar.elevation = 0f
+
+        // Reduce margin height of EngineView from the top for the toolbar
+        engineView.run {
```

Comment:
is this working? I had to change the previous way to `swipeRefresh` to not crash

Reply:
Yes, the view name is probably inconsistent. I pass in the swipeRefresh ViewGroup to the integration class. I'll re-test to verify.

Reply:
Ohh I see its being passed in but is actually swipeRefresh, gotcha :)

Reply:
I've renamed it to `engineLayout` to fit the context it's in. 👍